packages/html/src/events/touch.rs

```diff
@@ -1,17 +1,147 @@
 use dioxus_core::Event;
 
 pub type TouchEvent = Event<TouchData>;
-#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
-#[derive(Debug, Clone, PartialEq, Eq)]
 pub struct TouchData {
-    pub alt_key: bool,
-    pub ctrl_key: bool,
-    pub meta_key: bool,
-    pub shift_key: bool,
-    // get_modifier_state: bool,
-    // changedTouches: DOMTouchList,
```

Comment:
were you thinking of adding these back in a later PR or are you planning to add these here?

Reply:
I would like to add it back in this PR